firefox_ui_tests/remote/manifest.ini

```diff
@@ -1 +1,4 @@
+[DEFAULT]
+remote = true
```

Comment:
These types are just one per manifest right now, as long as they are let's not add them.